fail2ban/server/banmanager.py

```diff
@@ -26,6 +26,9 @@
 
 from threading import Lock
 
+import dns.exception
```

Comment:
>  > +import dns.exception
>    Informative message in the output, like "Banned ASN list: Failed to import
>    dnspython" ? 

something like

Banned ASN list:  [please install dnspython for this functionality]